pyfstat/helper_functions.py

```diff
@@ -429,6 +429,74 @@ def convert_array_to_gsl_matrix(array):
     return gsl_matrix
 
 
+def get_sft_as_numpy_array(sftfilepattern, fMin=None, fMax=None, constraints=None):
```

Comment:
This should be `get_sfts_as_numpy_array` or `get_sft_data_as_numpy_array`, as it handles multiple SFTs. (Note: the old `get_sft_array` is grammatically ok in this regard, as it can be read as "an array of SFT-type things", though on the other hand, technically it's a list of arrays...)

Reply:
PS: Actually this new one doesn't really do "as numpy array" either, it does a dict of arrays... `get_sfts_as_numpy_arrays`, or, since we don't really use non-nump arrays much, just `get_sfts_as_arrays`?

Reply:
Agree